tidb-cloud/changefeed-sink-to-cloud-storage.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Sink to Cloud Storage
+Summary: Learn how to create a changefeed to stream data from a TiDB Dedicated cluster to cloud storage, such as Amazon S3 etc.
+---
+
+# Sink to Cloud Storage
+
+This document describes how to stream data from a TiDB Dedicated cluster to cloud storage, such as Amazon S3 etc.
+
+
+> ## Limitations
+> 
+> - To use the Changefeed feature, please ensure that your TiDB cluster is running version v7.1.0 or a later version.
+> - Each dedicated cluster supports the creation of up to 5 changefeeds.
+> - As TiDB Cloud utilizes TiCDC for establishing changefeeds, it inherits the > identical [limitations of TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#unsupported-scenarios).
+> - Replicating data without a primary key or a non-null unique key constraint poses the risk of data redundancy, especially in certain retry scenarios where statements like INSERT and REPLACE are not reentrant.
```

Comment:
```suggestion
## Limitations

- To use Changefeed, ensure that your TiDB cluster is running version v7.1.0 or a later version.
- For each TiDB Cloud cluster, you can create up to 5 changefeeds.
- Because TiDB Cloud uses TiCDC to establish changefeeds, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#unsupported-scenarios).
- Replicating data without a primary key or a non-null unique key constraint poses the risk of data redundancy, especially in certain retry scenarios where statements like `INSERT` and `REPLACE` are not reentrant.
```